workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTesterWorkers.kt

```diff
@@ -0,0 +1,232 @@
+
+/**
+ * TODO update
+ * Specifies that this render pass is expected to run a particular worker.
+ *
+ * @param outputType TODO
+ * @param key The key passed to [runningWorker][com.squareup.workflow.RenderContext.runningWorker]
+ * when rendering this workflow.
+ * @param output If non-null, [WorkflowOutput.value] will be emitted when this worker is ran.
+ * The [WorkflowAction] used to handle this output can be verified using methods on
+ * [RenderTestResult].
+ * @param description Optional string that will be used to describe this expectation in error
+ * messages.
+ */
+/* ktlint-disable parameter-list-wrapping */
+@OptIn(ExperimentalStdlibApi::class)
+inline fun <PropsT, StateT, OutputT, RenderingT>
+    RenderTester<PropsT, StateT, OutputT, RenderingT>.expectWorkerOutputting(
+  outputType: KType,
+  key: String = "",
+  crossinline assertWorker: (Worker<*>) -> Unit = {},
+  output: WorkflowOutput<*>? = null,
+  description: String = ""
+): RenderTester<PropsT, StateT, OutputT, RenderingT> = expectWorker(
+/* ktlint-enable parameter-list-wrapping */
+    workerType = Worker::class.createType(listOf(KTypeProjection.covariant(outputType))),
+    key = key,
+    assertWorker = { assertWorker(it) },
+    output = output,
+    description = description.ifBlank { "worker outputting $outputType" + keyDescription(key) }
+)
+
+/**
+ * Specifies that this render pass is expected to run a particular worker.
+ *
+ * @param doesSameWorkAs Worker passed to the actual worker's
+ * [doesSameWorkAs][Worker.doesSameWorkAs] method to identify the worker. Note that the actual
+ * method is called on the worker instance given by the workflow-under-test, and the value of this
+ * argument is passed to that method – if you need custom comparison logic for some reason, use
+ * the overload of this method that takes a `matchesWhen` parameter.
+ * @param key The key passed to [runningWorker][com.squareup.workflow.RenderContext.runningWorker]
+ * when rendering this workflow.
+ * @param output If non-null, [WorkflowOutput.value] will be emitted when this worker is ran.
+ * The [WorkflowAction] used to handle this output can be verified using methods on
+ * [RenderTestResult].
+ * @param description Optional string that will be used to describe this expectation in error
+ * messages.
+ */
+/* ktlint-disable parameter-list-wrapping */
+@OptIn(ExperimentalStdlibApi::class)
+inline fun <
+    PropsT, StateT, OutputT, RenderingT, WorkerOutputT, reified WorkerT : Worker<WorkerOutputT>>
+    RenderTester<PropsT, StateT, OutputT, RenderingT>.expectWorker(
+  doesSameWorkAs: WorkerT,
+  key: String = "",
+  output: WorkflowOutput<WorkerOutputT>? = null,
+  description: String = ""
+): RenderTester<PropsT, StateT, OutputT, RenderingT> = expectWorker(
+/* ktlint-enable parameter-list-wrapping */
+    workerType = typeOf<WorkerT>(),
+    key = key,
+    assertWorker = { doesSameWorkAs.doesSameWorkAs(it) },
```

Comment:
Oops, this is the opposite of what this method is supposed to do according to the documentation.